# MSG HEARTBEAT — 0xFFFF — 65535

bool  system_error             # System error flag
bool  io_error                 # IO error flag
bool  swift_nap_error          # SwiftNAP error flag
uint8 sbp_minor_version        # SBP minor protocol version number
uint8 sbp_major_version        # SBP major protocol version number
bool  short_detected           # External antenna short flag
bool  external_antenna_present # External antenna present flag
